benchmarks/benchmarks.py

```diff
@@ -45,9 +45,7 @@ def __init__(self):
       self.callers.append(chr(ord('a')+i / 26) + chr(ord('a')+i % 26))
 
     for i in range(MAX_FILES):
-      filename = ""
-      for j in range(random.randint(1, 100)):
-        filename += '/' + chr(ord('a')+i / 26) + chr(ord('a')+i % 26)
+      filename = '/' + chr(ord('a')+i / 26) + chr(ord('a')+i % 26)
```

Comment:
Not sure if it affects benchmark time. I'll put it back, but with a reduced random range (less sub dir), what do you think? 
@grondinjc @maym2104